app/views/components/_deadline.html.erb

```diff
@@ -7,7 +7,7 @@
     </div>
     <div class="govuk-notification-banner__content">
       <p class="govuk-notification-banner__heading">
-        <%= raw inner_html.gsub(':remaining_days:', t('.remaining_days', count: remaining_days)) %>
+        <%= raw inner_html.gsub('{{remaining_days}}', t('.remaining_days', count: remaining_days)) %>
```

Comment:
@HanaBene toto pytal @jsuchal v komentari v #147 ale pozor, je to breaking change. Ak mas uz niekde pouzite `:remaining_days:` tak sa to mergnutim rozbije, kym sa to nezmeni na `{{remaining_days}}`